palette/support/src/dispatch.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017-2019 Parity Technologies (UK) Ltd.
+// Copyright 2019 Parity Technologies (UK) Ltd.
```

Comment:
no each file need to be have a license for the year it was created :D this can stay 2017-2019